include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_reduce.h

```diff
@@ -47,24 +47,37 @@ class __reduce_mid_work_group_kernel;
 template <typename... _Name>
 class __reduce_kernel;
 
+// Adjust number of sequential operations per work-item based on the vector size. Single elements are kept to
+// improve performance of small arrays or remainder loops.
+template <::std::uint8_t _VecSize, typename _Size>
+inline _Size
```

Comment:
I think `inline` isn't required for template functions

Reply:
I've removed the `inline` statement.